install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -677,6 +677,10 @@ data:
   disable-cnp-status-updates: "true"
 {{- end }}
 
+{{- if .Values.enableEgressGateway }}
```

Comment:
That's inconsistent with other names. We should probably go for `egressGateway.enabled=true` instead. Unless we're sure we won't ever need other subflags to `egressGateway` in which case just `egressGateway=true` is enough.